dexios-domain/Cargo.toml

```diff
@@ -15,7 +15,7 @@ authors = ["brxken128 <brxken128@tutanota.com"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-dexios-core = { path = "../dexios-core", version = "1.1.1" }
```

Comment:
it's a bad idea to choose `core` as a name. This will cause a conflict with [core](https://doc.rust-lang.org/stable/core/) in rust.

Reply:
I'd have to disagree. We use `domain`, and that could potentially conflict with the [domain](https://docs.rs/domain/latest/domain/) crate.

We use the `std` library, as we don't really have any use for `core` directly (direct usage seems to be for embedded devices? although I'm not too certain on that).

Reply:
it makes sense